server/services/group.js

```diff
@@ -19,6 +21,17 @@ function deleteGroup(groupName) {
   return Group.delete(groupName);
 }
 
+function canRemoveAdmin(groupName) {
```

Comment:
The naming here implies that the method will return a boolean, when it's actually returning a group or an error. Maybe `getGroupToRemoveAdmin` or something instead?